src/GraphQL/Builders/ConnectionBuilder.cs

```diff
@@ -202,10 +202,6 @@ private void CheckForErrors(ResolveConnectionContext<TSourceType> args)
             {
                 throw new ArgumentException("Cannot use `last` with unidirectional connections.");
             }
-            if (args.IsPartial && args.NumberOfSkippedEntries.HasValue)
-            {
-                throw new ArgumentException("Cannot specify `numberOfSkippedEntries` with partial connection resolvers.");
-            }
```

Comment:
Why?

Reply:
It will always be false, as IsPartial and NumberOfSkippedEntries is never initialized before this code is executed.  Therefore, the code has no purpose.

Reply:
Let's consider this change in another PR.

Reply:
I'll post a PR regarding removal of these properties after this one has been accepted, as it will need to be removed from the new interfaces also.